config.xml

```diff
@@ -1,49 +1,76 @@
-  <property name="ServerWebsiteURL" value="https://pcguru.sg"/>
-  <property name="TelnetPassword" value=""/>
-  <property name="TelnetPort" value="18081"/>
-  <property name="ZombiesRun" value="0"/>
-  <property name="TelnetEnabled" value="true"/>
-  <property name="AdminFileName" value="admins.xml"/>
-  <property name="SaveGameFolder" value="/home/steam/instances/BasicServer"/>
+  <property name="ServerPort" 				value="26900"/>				<!-- Port you want the server to listen on. -->
+  <property name="ServerIsPublic" 			value="true"/>				<!-- Should this server register to master server -->
+  <property name="ServerName" 				value="PCGURU.SG Kekistan No Raiding"/>		<!-- Whatever you want the name to be. -->
+  <property name="ServerPassword" 			value=""/>					<!-- Password to gain entry to the server -->
+  <property name="ServerMaxPlayerCount" 	value="32"/>					<!-- Maximum Concurrent Players -->
+  <property name="ServerReservedSlots"		value="6"/>					<!-- Out of the MaxPlayerCount this many slots can only be used by players with a specific permission level -->
+  <property name="ServerReservedSlotsPermission"	value="999"/>		<!-- Required permission level to use reserved slots above -->
+  <property name="ServerAdminSlots"			value="4"/>					<!-- This many admins can still join even if the server has reached MaxPlayerCount -->
+  <property name="ServerAdminSlotsPermission"	value="100"/>				<!-- Required permission level to use the admin slots above -->
+
+  <property name="ServerDescription" 		value="LULZ AND KEK - WE ARE ONE"/> <!-- Whatever you want the description to be. -->
+  <property name="ServerWebsiteURL" 		value="PCGURU.SG"/>					<!-- Website URL for the server -->
+  <property name="ServerDisabledNetworkProtocols"	value="UNET"/>			<!-- Networking protocols that should not be used. Separated by comma. Possible values: UNET, RakNet, SteamNetworking. Ex: "unet,steamnetworking" -->
+
+  <property name="GameWorld" 				value="Random Gen"/>			<!-- Navezgane, Random Gen -->
+  <property name="GameName" 				value="Kekistan1"/>			<!-- Whatever you want the game name to be THIS CONTROLS THE RANDOM GENERATION SEED -->
+  <property name="GameDifficulty" 			value="2"/>  				<!-- 0 - 5, 0=easiest, 5=hardest -->
+  <property name="GameMode"					value="GameModeSurvivalMP"/>	<!-- GameModeSurvivalMP, GameModeSurvivalSP (MP has land protection) -->
+
+  <property name="ZombiesRun"				value="0" />				<!-- 0 = default day/night walk/run, 1 = never run, 2 = always run -->
+  <property name="BuildCreate"				value="false" />				<!-- cheat mode on/off -->
+  <property name="DayNightLength"			value="60" />				<!-- real time minutes per in game day: 60 minutes -->
+  <property name="DayLightLength"			value="18" />				<!-- in game hours the sun shines per day: 18 hours day light per day-->
+  <property name="PlayerKillingMode"		value="3" />				<!-- Player Killing Settings (0 = No Killing, 1 = Kill Allies Only, 2 = Kill Strangers Only, 3 = Kill Everyone) -->
```

Comment:
Oh I see. Yeah that's fine. I can't always be around and players need to be able to kill trespassers. 

In the future if you ever successfully deploy Botman, I think they have a player plot system where strangers are not allowed to step into your territory. Then we can set to PVE.

Didn't set to value 2 also because.... friends killing friends seems to be a thing at night with some cliques lol.